build.gradle

```diff
@@ -28,6 +28,9 @@ subprojects {
 allprojects {
   repositories {
     mavenCentral()
+    maven {
```

Comment:
This seems to be working, the only missing piece is that we're using a BCEL release candidate.
Hopefully they will release the new version soon, then I'll remove this bit

Reply:
@gtoison Its ready to be removed now. I've merged in bcel 6.8.1